middleware/kubernetes/README.md

```diff
@@ -121,6 +121,42 @@ kubernetes coredns.local {
 	# Each line consists of the name of the federation, and the domain.
 	federation myfed foo.example.com
 	
+	# autopath [<host-domain>]
+	#
+	# Enables server side search path lookups for pods.  When enabled, coredns
+	# will identify search path queries from pods and perform the remaining path
+	# lookups on the pod's behalf.  The search path used mimics the resolve.conf
+	# search path deployed to pods. E.g.
+	#
+	#     search namespace.svc.cluster.local svc.cluster.local cluster.local
+	#
+	# <host-domain> is optional. If specified, coredns performs a lookup on this
```

Comment:
#1) its a matter of what kubernetes allows you to do, and we need to be able to mimic the search path.  I'll have see how these are configured in kubernetes, and if they allow more than one.  Or we can allow more than one regardless ...  cant hurt.

regarding #2... Yes perhaps. Cant hurt to allow the user to tweak the behavior.